index.js

```diff
@@ -55,6 +47,22 @@ module.exports = (api, options) => {
 
     if (lessThanWebpack5()) {
       webpackConfig.output.set("jsonpFunction", `webpackJsonp__${name}`);
+
+      webpackConfig.output.libraryTarget("umd");
```

Comment:
Let's not make this change at this time. UMD and System module formats aren't equivalent, so I'd consider this a breaking change. UMD would [require the SystemJS AMD extra](https://github.com/systemjs/systemjs/blob/main/docs/module-types.md#module-types), while System module would not. This would likely only make a difference with sharing dependencies, which is still an incomplete story when it comes to Vue 3 (see also https://github.com/systemjs/systemjs/issues/2272).

Reply:
To be clear, I think we should keep the UMD format for both versions until we can verify that Vue 3 with System format is workable. I'm happy to hear everyone else's thoughts.